modules/aws_vm/variables.tf

```diff
@@ -78,3 +78,7 @@ variable "os_disk_iops" {
 variable "subnet_id" {
   type = string
 }
+
+variable "enable_ebs_encryption" {
+  default = false
+}
```

Comment:
Provide a description here as to the purpose of the variable.

Reply:
Acknowledged. Description added.